### DISCUSSION TITLE
fluent-bit http output plugin generating many logs when output is connected

### DISCUSSION THREAD

**Discussion by nikhithakarennagari** [fluent/fluent-bit#6363] [General]
Hi @edsiper, @agup006, @patrick-stephens  @LionelCons Fluentbit http output plugin is generating many logs when connection is established with the output host. I am using the Fluentbit 1.9.9  version.

    Steps to reproduce the problem:
    Deploy fluent-bit as sidecar to any log producer and send the output events through http output plugin to any http host.
    Below continuous logs are seen when connection is established with the host.

{"timestamp":"2022-09-27T05:23:12.208+00:00","severity":"info","message":"[output:http:<host_name>] :port, HTTP status=200"}
{"timestamp":"2022-09-27T05:23:12.221+00:00","severity":"info","message":"[output:http:<host_name>] :port, HTTP status=200"}
{"timestamp":"2022-09-27T05:23:15.245+00:00","severity":"info","message":"[output:http:<host_name>] :port, HTTP status=200"}
{"timestamp":"2022-09-27T05:23:18.228+00:00","severity":"info","message":"[output:http:<host_name>] :port, HTTP status=200"}

Expected behavior
Single log message only should appear when connection is established until the disconnection happens instead of multiple logs which will spam the container.
Can you help me how to achieve this?

**Comment by agup006** [comment]
@nikhithakarennagari you may be able to set log_response_payload to false to stop these messages: https://docs.fluentbit.io/manual/pipeline/outputs/http

<img width="691" alt="image" src="https://user-images.githubusercontent.com/3642874/204456837-6a811c2a-de05-4843-834e-d8d9cec0b725.png">
